packages/core/admin/ee/admin/pages/SettingsPage/pages/Webhooks/EditView/components/EventTable/EventTableEE.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import EventTable from '../../../../../../../../../admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/Events';
+
+const events = {
+  'review-workflows': ['workflows.updateEntryStage'],
+};
+
+// TODO: extend this to support review workflow events once the BE logic is ready
+// const getHeaders = () => {
+//   return [{ id: 'review-workflows.updateEntryStage', defaultMessage: 'Stage Change' }];
+// };
+
+export function EventTableEE() {
+  return (
+    <EventTable.Root
+      renderChildren={({ isDraftAndPublish }) => {
+        return (
+          <>
+            <EventTable.Headers isDraftAndPublish={isDraftAndPublish} />
+            <EventTable.Body isDraftAndPublish={isDraftAndPublish} extraEvents={events} />
```

Comment:
I think we can improve this further (maybe discuss it with Josh next week), but I think to get this merged it is ok for now.